plugins/search-backend-node/src/engines/LunrSearchEngine.test.ts

```diff
@@ -0,0 +1,225 @@
+/*
+ * Copyright 2021 Spotify AB
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { getVoidLogger } from '@backstage/backend-common';
+import { LunrSearchEngine } from '.';
+import { SearchEngine } from '../';
```

Comment:
Just a reminder that we don't want to import from dots like this. It leads to confusing circularity. The first one would be `from './LunrSearchEngine'` and the second one `from '../types'` if i read things correctly